### PR TITLE
Log runner that is being executed

### PR DESCRIPTION
**Pull request by nick-stroud**
I have wanted this information several times in the past. It will help to make it clear which runner logs are coming from as well as confirming that all runners have executed. 

### Submission Checklist

* [x] Have you installed and run this change against pre-commit? (`pre-commit
  install`)
* [x] Are all tests passing? (`make tests`)
* [ ] Have you written unit tests to cover this change?
* [x] Is unit test coverage still above 80%?
* [x] Have you updated all applicable documentation?
* [x] Have you followed the guidelines in our Contributing document?

### CODE REVIEW

File: modules/scripts/startup-script/templates/startup-script-custom.tpl [hunk 1]
```diff
@@ -27,10 +27,12 @@ stdlib::runner() {
...
 
+  echo "start executing runner: $filename"
   case "$1" in
     ansible-local) stdlib::run_playbook "$destpath/$filename" "$args";;
     shell) . $destpath/$filename $args;;
   esac
+  echo "finished executing runner: $filename"
   
   exit_code=$?
   stdlib::info "=== $object finished with exit_code=$exit_code ==="
```
Comment (heyealex) [inline modules/scripts/startup-script/templates/startup-script-custom.tpl:38]: This line already announces when the runner has completed. I'd stick with this format for the "start executing runner" line as well.

Comment (cboneti) [reply modules/scripts/startup-script/templates/startup-script-custom.tpl:38]: What is the difference between using echo here, vs stdlib::info? Shouldn't the latter be preferred?

Comment (nick-stroud) [reply modules/scripts/startup-script/templates/startup-script-custom.tpl]: I honestly had never noticed that line in the logs. 

I have:
- removed the finished line
- switched to using stdlib::info
- updated start line format to look similar to finished

We can either merge or close. I think it is marginally better to also print the begining to make it clear the runner has started.

Comment (heyealex) [reply modules/scripts/startup-script/templates/startup-script-custom.tpl]: It was added recently, with the switch away from source. 
I like including the begin line as well, if a runner stalls you don't have to go back to the blueprint to figure out which one was active currently.

**Review by heyealex — APPROVED** [review]

**Pull request merged**